roles/postgresql/tasks/backup_db.yml

```diff
@@ -0,0 +1,13 @@
+---
+# back up the database to /tmp/ on the app's VM (not the database VM)
+- name: backup postgres database
```

Comment:
How hard to skip the backup if create database resulted in a brand new db?

Reply:
hmm, great question. i think it would just be registering the output of the step that creates the db and checking if `changed: true`, maybe? that's how i would expect the ansible `postgresql` module to work, anyway.

Reply:
that's my guess too

Reply:
tracked on #53.